components/resources/brave_vpn_strings.grdp

```diff
@@ -4,6 +4,10 @@
     Brave Firewall + VPN
   </message>
 
+  <message name="IDS_BRAVE_VPN_PANEL_NAME" desc="The name of VPN panel">
+    Brave VPN
```

Comment:
nit:"Brave VPN UI", so users can see that this is the renderer process for VPN and not the VPN itself

Reply:
@petemill How about `Brave VPN Panel UI`?

Reply:
Sure!

Reply:
Done.